packages/database/src/migrations/20210603022946-AddFijiSupplyChainHeirarchySubDistricts-modifies-data.js

```diff
@@ -5,31 +5,39 @@ var type;
 var seed;
 
 /**
-  * We receive the dbmigrate dependency from dbmigrate initially.
-  * This enables us to not have to rely on NODE_PATH.
-  */
-exports.setup = function(options, seedLink) {
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
   dbm = options.dbmigrate;
   type = dbm.dataType;
   seed = seedLink;
 };
 
-exports.up = function(db) {
+exports.up = function (db) {
   return db.runSql(`
     update "entity_hierarchy" 
-    set "canonical_types" = '{country,district,sub_district,facility}' 
+    set "canonical_types" = '{country,district}' 
```

Comment:
Just confirming, we no longer need `sub_district` here?

Reply:
Sub_district and facility are specified via entity_relations rather than parent_id